_includes/cloudcode/cloud-code.md

```diff
@@ -598,6 +598,46 @@ Parse.Cloud.afterLogout(async request => {
 
 # LiveQuery Triggers
 
+## beforeConnect
+
+*Available only on parse-server cloud code starting 4.3.0*
+
+You can run custom Cloud Code before a user attempts to connect to your LiveQuery server. You can do this with the `beforeConnect` method. For instance, this can be used to only allow users that have logged in to connect to the LiveQuery server. 
```

Comment:
```suggestion
You can run custom Cloud Code before a user attempts to connect to your LiveQuery server with the `beforeConnect` method. For instance, this can be used to only allow users that have logged in to connect to the LiveQuery server. 
```